src/drivers/intel/dmic.c

```diff
@@ -134,9 +134,8 @@ static const uint32_t coef_base_a[4] = {PDM0_COEFFICIENT_A, PDM1_COEFFICIENT_A,
 static const uint32_t coef_base_b[4] = {PDM0_COEFFICIENT_B, PDM1_COEFFICIENT_B,
 	PDM2_COEFFICIENT_B, PDM3_COEFFICIENT_B};
 
-/* Global configuration request for DMIC, need to use uncached address to access them */
-static SHARED_DATA struct sof_ipc_dai_dmic_params *dmic_prm[DMIC_HW_FIFOS];
-static SHARED_DATA int dmic_active_fifos_mask;
+/* Global configuration request and state for DMIC, need to use uncached address to access them */
+static SHARED_DATA struct private_shared dmic_shared;
```

Comment:
code gets copied around (TM). Which means other drivers might copy this verbatim with the same `private_shared` and `private_nonshared` names, which makes grepping more difficult. Maybe better use namespaced names too, e.g. `dmic_private_shared`